src/entities/candidate/candidate.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Param, ParseIntPipe, Req } from '@nestjs/common';
+import { Candidate } from './candidate.entity';
+import { CandidateService } from './candidate.service';
+
+@Controller('api/v1/candidates')
+export class CandidateController {
+  constructor(private candidateService: CandidateService) {}
```

Comment:
Please, Make candidateService property readonly in the constructor.

Reply:
Done, also fixed same troubles in another controllers and services